views/pages/funding/guidance/logos.njk

```diff
@@ -68,26 +68,28 @@
     </div>
 {% endmacro %}
 
+{# whitespace/linebreaks here render in the href attribute :( #}
+{% macro logoPath(locale, filename, extension, suffix = false) %}{% set l = 'logos/' + locale + '/' + filename %}{% if suffix %}{% set l = l + suffix %}{% endif %}{% set l = l + '.' + extension %}{{ l | getImagePath }}{% endmacro %}
```

Comment:
~What happens if you pass the function through `trim`, e..g `{{logoPath(logoLocale, logoFilename, 'png') | trim }}`. Does that help with this issue? Or maybe https://mozilla.github.io/nunjucks/templating.html#whitespace-control~

Checked the code out and doing using trim seemed to work:

```
{% macro logoPath(locale, filename, extension, suffix = false) %}
    {% set l = 'logos/' + locale + '/' + filename %}
    {% if suffix %}
        {% set l = l + suffix %}
    {% endif %}
    {% set l = l + '.' + extension %}
    {{ l | getImagePath | trim }}
{% endmacro %}
…
{{ logoPath(logoLocale, logoFilename, 'jpg', '-large') | trim }}
```